src/main/java/com/fnz/TimeTracking/model/Permission.java

```diff
@@ -0,0 +1,22 @@
+package com.fnz.TimeTracking.model;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Entity
+public class Permission {
+
+    @Id
+    @GeneratedValue
+    private long id;
+    private LocalDateTime heureSortie;
+    private LocalDateTime heureRetour;
+
+    @ManyToOne
```

Comment:
This relation is wrong because the User has direct connection with Conge, so this must be removed.

instead, add A OneToOne relation with Permission > Conge